src/walle-lib/Digging_Robot.hpp

```diff
@@ -0,0 +1,20 @@
+//
+// Created by tonys on 29/11/2024.
+//
+
+#ifndef DIGGING_ROBOT_HPP
+#define DIGGING_ROBOT_HPP
+
+#include "Robot.hpp"
+
+class Digging_Robot : public Robot
```

Comment:
The name of the class Digging_Robot does not seem to follow the same naming convention as the IrrigationRobot. Please unify.